internal/storetesting/charm-repo/quantal/multi-series/metadata.yaml

```diff
@@ -0,0 +1,28 @@
+name: multi-series
+summary: multi-series test charm
+description: Test charm that supports a number of series
+series:
+ - trusty
+ - utopic
+ - vivid
+ - wily
+provides:
```

Comment:
yes, there are some upload testing checks that test that these are specified as this.